analyses/oncoprint-landscape/run-oncoprint.sh

```diff
@@ -6,6 +6,9 @@
 set -e
 set -o pipefail
 
+# Print oncoprints by broad histology - will not by default
+HISTOLOGY=${RUN_BROAD_HISTOLOGY:-0}
```

Comment:
I think we can take this logic for skipping splitting up by broad histology out. We know we probably want to break the plots up into separate panels. The way you've altered `01-plot-oncoprint.R` means one could just run without `--broad_histology` and get plots for the entire cohort if we wanted!